README.md

```diff
@@ -95,125 +123,27 @@ query CityById{
 }
 ```
 
----
-# README
-
-Welcome to [RedwoodJS](https://redwoodjs.com)!
-
-> **Prerequisites**
->
-> - Redwood requires [Node.js](https://nodejs.org/en/) (>=14.19.x <=16.x) and [Yarn](https://yarnpkg.com/) (>=1.15)
-> - Are you on Windows? For best results, follow our [Windows development setup](https://redwoodjs.com/docs/how-to/windows-development-setup) guide
+## Running Edge Functions in GraphQL
 
-Start by installing dependencies:
+Add the following to your `netlify.toml` file:
 
 ```
-yarn install
+[[edge_functions]]
+path = "/.redwood/functions/graphql"
+function = "geolocate"
 ```
 
-Then change into that directory and start the development server:
+Adding this to the `.toml` file means that when a request is made to the `path`, it will invoke the `geolocate` edge function as part of the request.
```

Comment:
[dust] Capitalize Edge Function for consistency.
```suggestion
Adding this to the `.toml` file means that when a request is made to the `path`, it will invoke the `geolocate` Edge Function as part of the request.
```